source/_posts/2021-03-03-release-20213.markdown

```diff
@@ -2534,7 +2534,7 @@ The following integrations are no longer available as of this release:
 [geniushub docs]: /integrations/geniushub/
 [gios docs]: /integrations/gios/
 [github docs]: /integrations/github/
-[google docs]: /integrations/google/
+[google docs]: /integrations/calendar.google/
```

Comment:
This is generated... we should not correct this here, instead, we should ensure the link works (e.g., by creating a redirect).

Reply:
I agree. I'll make a redirect.

Reply:
Done.